synapse/appservice/api.py

```diff
@@ -262,6 +268,16 @@ async def push_bulk(
                 }
             )
 
+        if service.msc3202_transaction_extensions:
+            if one_time_key_counts:
+                body[
+                    "org.matrix.msc3202.device_one_time_key_counts"
+                ] = one_time_key_counts
+            if unused_fallback_keys:
+                body[
+                    "org.matrix.msc3202.device_unused_fallback_keys"
```

Comment:
Having tried to play with this just now, I think this is wrong. The MSC states [`device_unused_fallback_key_types`](https://github.com/matrix-org/matrix-spec-proposals/blob/travis/msc/otk-dl-appservice/proposals/3202-encrypted-appservices.md#proposal)